app/models/cashbox.rb

```diff
@@ -0,0 +1,7 @@
+class Cashbox < ActiveRecord::Base
+  has_many :deposits, class_name: 'Payment', foreign_key: 'destination_id'
+  has_many :withdrawls, class_name: 'Payment', foreign_key: 'source_id'
```

Comment:
@b264 I'm already using  two  two foreign ids-- source and destination on payment table via this [migration](https://github.com/AgileVentures/osra/pull/441/files#diff-022e8f544d24cf4c36d9116d2fd0c1c3) 

@NikitaAvvakumov I guess that payment should be a positive value between source and destination for example if we have a transaction/payment between two cashbox and we set a negative value (in this case this value represents a withdrawal from a cashbox which is negative value but it's in the same time a deposit to the destination cashbox which should be positive)
That's why I opted for two separate associations to keep positive value and use FK on payment model ( source and destination)